frontend/src/app/modules/interventions/list/table/table.component.html

```diff
@@ -1,46 +1,48 @@
-<div class="mat-elevation-z4">
-    <table mat-table [dataSource]="dataSource">
-
-        <!-- Position Column -->
-        <ng-container matColumnDef="position">
-            <th mat-header-cell *matHeaderCellDef> Numer </th>
-            <td mat-cell *matCellDef="let element"> {{element.position}} </td>
-        </ng-container>
-
-        <!-- Name Column -->
-        <ng-container matColumnDef="name">
-            <th mat-header-cell *matHeaderCellDef> Imię i nazwisko </th>
-            <td mat-cell *matCellDef="let element"> {{element.fullName}} </td>
-        </ng-container>
-
-        <!-- Status Column -->
-        <ng-container matColumnDef="status">
-            <th mat-header-cell *matHeaderCellDef> Status </th>
-            <td mat-cell *matCellDef="let element"> {{element.status}} </td>
-        </ng-container>
-
-        <!-- Phone Column -->
-        <ng-container matColumnDef="phone">
-            <th mat-header-cell *matHeaderCellDef> Telefon </th>
-            <td mat-cell *matCellDef="let element"> {{element.phone}} </td>
-        </ng-container>
-
-        <!-- Date Column -->
-        <ng-container matColumnDef="date">
-            <th mat-header-cell *matHeaderCellDef> Data dodania </th>
-            <td mat-cell *matCellDef="let element"> {{element.creationDate | date: 'medium' }} </td>
-        </ng-container>
-
-        <!-- Description Column -->
-        <ng-container matColumnDef="description">
-            <th mat-header-cell *matHeaderCellDef> Opis </th>
-            <td mat-cell *matCellDef="let element" title="{{element.description}}"> {{element.description}} </td>
-        </ng-container>
-
-        <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
-        <tr mat-row *matRowDef="let element; columns: displayedColumns;" (click)='showDetails(element)'></tr>
-
-    </table>
-    <mat-paginator [pageSize]="10" [pageSizeOptions]="[10, 25, 50, 100]">
-    </mat-paginator>
+<div class="mat-elevation-z8">
+  <mat-table mat-table [dataSource]="dataSource">
```

Comment:
Since we are now using material element tags e.g `<mat-table></mat-table>` , `<mat-cell></mat-cell>`  etc, could you please check if we still need the directives with same text as tag names? 
It looks pretty weird to write `<mat-table mat-table></mat-table>` and I suspect that the directives might not be needed anymore :)

I believe that the approach for enhancing normal html elements with directives is derived from accessability and semantic markup paradigms, where screen readers could still understand what is happening by following standard html tags, but adding the directive allows enhancing the element by Angular.

I am not especially keen on one or the other approach, both seem good to me, but if possible I'd rather not mix them to keep the markup concise :)

Reply:
Directives are not needed indeed. I've removed them 🔪 
https://material.angular.io/components/table/overview#tables-with-code-display-flex-code-

Also, this table which is based on custom HTML elements is still accessible, since `mat-table` generates ARIA role attributes. 🙂
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Gridcell_role